src/index.scss

```diff
@@ -1,4 +1,34 @@
 :root {
+  --screen-mobile-s: 320px;
+  --screen-mobile-m: 375px;
+  --screen-mobile-l: 425px;
+  --screen-tablet: 768px;
+  --screen-laptop: 1024px;
+  --laptop-l: 1440px;
```

Comment:
use the same naming for variables 
ex:
--screen-laptop
--screen-l
--screen-xl